packages/ui/src/components/molecules/Toast/styles.scss

```diff
@@ -32,7 +32,7 @@
   // Title
   --fs-toast-title-size                   : var(--fs-text-size-body);
   --fs-toast-title-weight                 : var(--fs-text-weight-bold);
-  --fs-toast-title-line-height            : var(--fs-scale);
+  --fs-toast-title-line-height            : 1.2;
```

Comment:
suggestion: @renatamottam I noticed that we are using this same `line-height:1.2` value for other components too, WDYT about adding it as a global token?

Reply:
I love this idea, I'm gonna create a task for it on our backlog ✨